independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/compress/QuarkusProjectCompress.java

```diff
@@ -17,34 +16,33 @@
 
 public final class QuarkusProjectCompress {
 
-    private static final List<String> EXECUTABLES = ImmutableList.of(
+    private static final List<String> EXECUTABLES = Collections.unmodifiableList(Arrays.asList(
             "gradlew",
             "gradlew.bat",
             "mvnw",
-            "mvnw.bat");
+            "mvnw.bat"));
 
-    @VisibleForTesting
+    // Visible for testing
     static final int DIR_UNIX_MODE = UnixStat.DIR_FLAG | UnixStat.DEFAULT_DIR_PERM;
 
     private QuarkusProjectCompress() {
     }
 
-    public static void zip(final QuarkusProject quarkusProject, final Path targetZipPath, final boolean includeProjectFolder)
+    public static void zip(final Path quarkusProjectFolderPath, final Path targetZipPath, final boolean includeProjectFolder)
```

Comment:
Folder? Is that a Windows jargon? :) I think all the Java API I've seen uses "directory" or "dir".

Reply:
oh boy, I've used folder at many places 😬
let me rename it all 😅